### PR TITLE
[DOC] examples for min() function

### PR DESCRIPTION
**Pull request by samukweku**
WIP #2604 
- Examples for the min() function

### CODE REVIEW

File: src/core/expr/head_reduce_unary.cc [hunk 1]
```diff
@@ -755,6 +755,84 @@ except: TypeError
...
+    4,2,6
+    5,2,2
+    6,3,3
+    7,3,22
+    8,3,1
+
+
+Get the minimum from column B::
+
+    df[:, dt.max(f.B)]
```
Comment (st-pasha) [inline src/core/expr/head_reduce_unary.cc:789]: `dt.min`

**Review by st-pasha — APPROVED** [review]

**Comment by samukweku** [comment]
issues with my repo locally. I will delete this and create a new PR.

**Comment by st-pasha** [comment]
Btw, you can create PRs directly in this repository, without having to maintain a fork. Should be easier that way.

**Comment by samukweku** [comment]
that's a new one for me @st-pasha ; how do I go about that?

**Comment by st-pasha** [comment]
```
git clone https://github.com/h2oai/datatable
cd datatable
git checkout -b so/mybranch
git commit
git push 
```
then open https://github.com/h2oai/datatable/pulls page and press the "create new PR" button, it should be pre-set to the branch that you just pushed

**Pull request closed without merging**